bundle/tests/job_cluster_key/databricks.yml

```diff
@@ -0,0 +1,27 @@
+bundle:
+  name: job_cluster_key
+
+workspace:
+  host: https://acme.cloud.databricks.com/
+
+targets:
+  default:
+    resources:
+      jobs:
+        foo:
+          name: job
+          tasks:
+          - task_key: test
+            job_cluster_key: key
```

Comment:
You could define this one globally and only configure the job cluster in the development target to achieve the same.